scripts/deploy.sh

```diff
@@ -48,8 +49,13 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   if [ -n "${DATA_DISK_DEVICE}" ]; then
     export BOOTSTRAP_OPTS="${BOOTSTRAP_OPTS} bootstrap_host_data_disk_device=${DATA_DISK_DEVICE}"
   fi
+
   # force the deployment of haproxy for an AIO
   export DEPLOY_HAPROXY="yes"
+
+  # Enable host security hardening for AIO builds
+  export DEPLOY_HARDENING="yes"
```

Comment:
I think this should be removed, I see no reason to force an AIO to deploy this playbook. As it stands the feature is optional and so the AIO should be able to take advantage of that configurability.